README.md

```diff
@@ -1,2 +1,13 @@
 # HW01
-Edit the readme
+## Random edit
+r <- 8
+r + r
+
+## About me
+
+* I am currently living in Oregon.
+* I will be moving to Chicago in September.
+* I just finished reading *An American Marriage*; I recommend it.
+* I really enjoyed watching ATLA last month; it was **really good**.
```

Comment:
Is this Avatar: The Last Airbender? I watched it recently too!

Reply:
Yup!!